docs/websocket.md

```diff
@@ -307,3 +307,283 @@ eg:
    }
 }
 ```
+
+### Orders
+
+The orders channel allows clients to subscribe to orders and fills pertaining to the wallet address.
+
+#### Subscribing
+
+To subscribe, send:
+
+```json
+{
+  "type": "subscribe",
+  "channel": "orders",
+  "id": "0x014be43bf2d72a7a151a761a1bd5224f7ad4973c"
+}
+```
+
+|Field Name|JSON type|Description|
+|----------|---------|-----------|
+|type|string|Must be set to "subscribe"|
+|channel|string|Must be set to "orders"|
+|id|string|The wallet address to listen to|
+
+#### Initial Response
+
+The initial response will be all the user's open and pending orders, inside the
+`contents` field.
+
+```json
+{
+  "type": "subscribed",
+  "connection_id": "a3f64f25-4744-46f8-8456-f456f75616b2",
+  "message_id": 1,
+  "channel": "orders",
+  "id": "0x014be43bf2d72a7a151a761a1bd5224f7ad4973c",
+  "contents": {
+    "orders": [
+      {
+        "uuid": "210ab138-017d-4983-8d87-2ca6ff0955c8",
+        "id": "0x119f4e6e453a90c0eb64f6c2d3015386c4d201e89ea35a35495cba4c7edb4665",
+        "createdAt": "2020-01-14T21:56:18.765Z",
+        "status": "OPEN",
+        "accountOwner": "0x014be43bf2d72a7a151a761a1bd5224f7ad4973c",
+        "accountNumber": "0",
+        "orderType": "LIMIT",
+        "fillOrKill": false,
+        "postOnly": true,
+        "market": "DAI-USDC",
+        "side": "BUY",
+        "baseAmount": "20000000000000000000",
+        "quoteAmount": "1000000",
+        "filledAmount": "0",
+        "price": "0.00000000000005",
+        "cancelReason": null
+      }
+    ]
+  }
+}
+```
+
+#### Updates
+
+Updates to the orders are posted on the channel. If an order is filled, the corresponding fills will also be
+sent on the channel. eg:
```

Comment:
Just mention that orders / fills are in the same schema as the V2 http API

Reply:
The v2 http schema for orders and fills is not on master yet, should I add it.

Reply:
Phil's about to add it, so yeah we should

Reply:
added